docs/content/docs/how-to/writing-tables.md

```diff
@@ -87,3 +87,66 @@ INSERT OVERWRITE MyTable PARTITION (key1 = value1, key2 = value2, ...) SELECT ..
 {{< /tab >}}
 
 {{< /tabs >}}
+
+## Purging tables
+
+You can use `INSERT OVERWRITE` to purge tables by inserting empty value.
+
+{{< tabs "purge-tables-syntax" >}}
+
+{{< tab "Flink" >}}
+
+```sql
+INSERT OVERWRITE MyTable SELECT * FROM MyTable WHERE false
+```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Purging a Partition
+
+Particularly, you can use `INSERT OVERWRITE` to purge data of a partition by inserting empty value to the partition:
+
+{{< tabs "purge-partition-syntax" >}}
+
+{{< tab "Flink" >}}
+
+```sql
+INSERT OVERWRITE MyTable PARTITION (key1 = value1, key2 = value2, ...) SELECT selectSpec FROM MyTable WHERE false
+```
+
+{{< /tab >}}
+
+{{< /tabs >}}
+
+{{< hint warning >}}
+
+The `selectSpec` should includes all the columns in the table except the partition columns behind PARTITION.
```

Comment:
Good, we can copy this line to above `Overwriting a Partition` too.

Reply:
This hint is not compatible with `Overwriting a Partition` section because here it selects data from the same table but in `Overwriting a Partition` section it can select data from any table.